pkg/controller/vpc_dns.go

```diff
@@ -294,12 +294,17 @@ func (c *Controller) genVpcDNSDeployment(vpcDNS *kubeovnv1.VpcDns) (*v1.Deployme
 		klog.Errorf("failed to parse coredns template file, %v", err)
 		return nil, err
 	}
+	vpcDNSCorefile := vpcDNS.Spec.Corefile
+	if vpcDNSCorefile == "" {
+		vpcDNSCorefile = "vpc-dns-corefile"
```

Comment:
To maintain compatibility, we should avoid rendering `vpcDNSCorefile` if it is empty.